spec/requests/api/v1.0/service_plans_spec.rb

```diff
@@ -19,6 +19,6 @@
   include_examples(
     "v1x0_test_index_and_subcollections",
     "service_plans",
-    ["service_instances"],
+    [],
```

Comment:
Is this empty array necessary or can we just remove this as an argument??

Reply:
It's the necessary parameter, means the `subcollections` is empty